microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -46,18 +46,18 @@ To use the Microsoft Edge Add-ons API, you need to enable the API for your proje
 
 1. On the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes to finish.
 
-   ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)
-
- > [!IMPORTANT]
-> You can generate multiple client secrets for your client ID.  For example, you can create multiple secrets for multiple projects.
+1. The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client secret**, **Expiry date**, and **Access token URL** are now displayed on the Publish APIs page:
 
-   The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client secret**, **Expiry date**, and **Access token URL** are now displayed on the Publish APIs page.
+   ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)
 
 1. Write down the **Client ID**, **Client secret** and the **Access token URL**.  You'll use these values in the next step, to get an access token.
 
 > [!IMPORTANT]
 > Be sure to write down the client secret now, because it's only visible immediately after enabling or renewing the API (that is, after creating API credentials). This particular secret isn't shown again.
 
+Note that you can generate multiple client secrets for your client ID.  For example, you can create multiple secrets for multiple projects.
```

Comment:
```suggestion

###### Generating multiple client secrets

You can generate multiple client secrets for your client ID.  For example, you can create multiple secrets for multiple projects.
```
instead of "Note that", could add a heading, to provide a skimmable label